manifests/master.pp

```diff
@@ -328,55 +328,32 @@
 
     $_conf_base = dirname($confdir)
 
-    file { $_conf_base:
-      ensure => 'directory',
-      owner  => 'root',
-      group  => 'puppet',
-      mode   => '0640'
-    }
-
-    file { $confdir:
-      ensure => 'directory',
-      owner  => 'root',
-      group  => 'puppet',
-      mode   => '0640'
+    file { '/var/log/puppetserver':
```

Comment:
When is this a thing? It is not part of the SIMP default load or created on a running system.

Reply:
I had to create it for some reason - I don't remember at the moment. It can be removed.